src/mono/browser/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = false
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
+insert_final_newline = true
+indent_style = tabs
```

Comment:
we don't use tabs in TS, do we really want to merge this file ?

Reply:
oh, that looks wrong! good catch!